syncer/sharding_group.go

```diff
@@ -380,6 +381,13 @@ func (sg *ShardingGroup) FlushData(targetTableID string) ([]string, [][]interfac
 
 // GenTableID generates table ID
 func GenTableID(schema, table string) (ID string, isSchemaOnly bool) {
+	log.L().Warn("hotfix, will change schema and table name to lower",
```

Comment:
shall we only output this warn log when the lower value is different from the origin value?

Reply:
https://github.com/pingcap/dm/pull/1641/commits/99de7c5b600dac1909b4e9f2831c3f6ac622defe